### ISSUE TITLE
Policies page (Refinery)

### ISSUE THREAD

**Issue by DeniseColbert** [FreeUKGen/FreeBMD2#537]
To house all policies and Terms and Conditions.

Footer links could be:

Privacy Notice
Accessibility statement
All policies

Contact us
Donate
******************************
[And then space for something else?]

**Comment by AlOneill** [comment]
Apart from these, what else should we include:
* Privacy notice
* Accessibility statement (rather than policy — I cannot edit the top information here or assign myself)
* Terms and conditions

**Comment by AlOneill** [comment]
* Cookie policy

**Comment by AlOneill** [comment]
[DRAFT page](https://test3.freebmd.org.uk/cms/draft-policies-notices-and-statements) created

**Comment by AlOneill** [comment]
Cannot move to 'In progress'

**Comment by DeniseColbert** [comment]
> Apart from these, what else should we include:
> 
> * Privacy notice
> * Accessibility statement (rather than policy — I cannot edit the top information here or assign myself)
> * Terms and conditions

Also, Safeguarding Policy @AlOneill 

I've assigned you, moved to in Progress, and changed the description to reflect 'statement' rather than 'policy'

**Comment by AlOneill** [comment]
@DeniseColbert 
Thank you. Any idea why I've suddenly lost the ability to make the changes myself? It's a nuisance.

**Comment by DeniseColbert** [comment]
Security reporting policy also
